concrete/src/Notification/Alert/Filter/FilterInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Concrete\Core\Notification\Alert\Filter;
+
+use Concrete\Core\Notification\Alert\AlertList;
+
+interface FilterInterface
+{
+
+    public function getName();
```

Comment:
Marking the `/** @return string */` for this would allow IDEs to understanding the input/output and hint properly